distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -387,22 +402,22 @@ logging:pattern=%date{ISO8601} - %-25logger: %msg%n
 
 ############################ GCal Persistence Service #################################
 #
-# the username and password for Google Calendar Account
-#gcal-persistence:username=
-#gcal-persistence:password=
-
-# the url of the calendar feed
-#gcal-persistence:url=
+# The following parameters are used to configure the GCal Persistence binding.
+# Authentication to the Google Calendar API is performed using the GCal binding's
+# OAuth authentication mechanism. 
+#
+# This is the name you gave to your Google Calendar. GCal Persistence will persist 
+# to this calendar (required)
+#gcal-persistence:calendar_name=
 
-# the offset (in days) new calendar entries will be created in future (optional,
+# The offset (in days) new calendar entries will be created in future (optional,
 # defaults to 14)
 #gcal-persistence:offset=
 
-# the base script which is written to the newly created Calendar-Events by
+# The base script which is written to the newly created Calendar-Events by
 # the GCal-based presence simulation. It must contain two format markers '%s'.
 # The first marker represents the Item to send the command to and the second
-# represents the State (optional, defaults to
-# '> if (PresenceSimulation.state == ON) sendCommand(%s,%s)')
+# represents the State (optional, defaults to 'send %s %s')
 #gcal-persistence:executescript=
```

Comment:
I left it in because it's still pulled from the config on line 116, and is used on line 166, in GCalPersistenceService.java